autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -48,6 +49,7 @@
     CAT=CatboostModel,
     NN=TabularNeuralNetModel,
     LR=LinearModel,
+    FASTAINN=NNFastAiTabularModel,
```

Comment:
Consider just using short name `FASTAI` since I think it's their only tabular data model

Reply:
Updated.
Note: they have pretty good NLP models, which can be added once transfer learning loops are available.